### PR TITLE
Warnings fix

### PR DESCRIPTION
**Pull request by anilyil**
## Purpose
Updated the code to remove the compiler warnings
Updated gitignore for the input files
Fixed a few use only statements where we had both a bare import and specific imports
Improved syntax of the input file instructions

## Type of change
- Bugfix (non-breaking change which fixes an issue)
- Code style update (formatting, renaming)

## Checklist
- [x] I have run unit and regression tests which pass locally and on pleiades with my changes

### CODE REVIEW

**Comment by nwu63** [comment]
There are additional warnings in the complex build. I will attempt to address them tonight.

File: src/overset/zipperMesh.F90 [hunk 1]
```diff
@@ -1231,14 +1231,13 @@ end function checkDeviation
...
 
-    use communication
     !use oversetData
     use constants
     use blockPointers
     use utils, only : setPointers, setBCPointers
     use BCPointers, only : xx
     use sorting, only : famInList
-    use communication, only : myid, adflow_comm_world
+    use communication, only : myid, adflow_comm_world, nProc
```
Comment (joanibal) [inline src/overset/zipperMesh.F90:1240]: Why was `nProc` added?

**Comment by anilyil** [comment]
@sseraj check out issue #53. I explain the warnings not addressed by this PR in there

**Comment by anilyil** [comment]
yeah, please add the additional warnings you found to issue #53 if its not there already. I think the stuff you mentioned is all related to the warnings that come up after f2py, I tried to group all of those together in item 4 in that issue.

**Comment by joanibal** [comment]
Ssooo, are we good to pull this in?

**Comment by sseraj** [comment]
@joanibal I would say yes if you're good with the changes by @nwu63

**Comment by nwu63** [comment]
I've added all the changes I wanted. If you are good with it then please approve and pull it in, thanks.

**Pull request merged**